packages/types/src/mtv-room-websockets.ts

```diff
@@ -67,4 +72,6 @@ export interface MtvRoomServerToClientEvents {
     FORCED_DISCONNECTION: () => void;
     JOIN_ROOM_CALLBACK: (state: MtvWorkflowState) => void;
     CHANGE_EMITTING_DEVICE_CALLBACK: (state: MtvWorkflowState) => void;
+    SUGGEST_TRACKS_CALLBACK: (state: MtvWorkflowState) => void;
```

Comment:
Shall we rename the SUGGEST_TRACKS_CALLBACK to something like SUGGEST_TRACKS_LIST_UPDATE as it's client socket event related.
Also ACKNOWLEDGE_TRACKS_SUGGESTION might be more the SUGGEST_TRACKS_CALLBACK as it's related to the client socket event SUGGEST_TRACKS

Reply:
Done